### PR TITLE
Feature/45 slc build

### LINKED ISSUE #45: Move SLC build from TE to PEC

### PR DESCRIPTION
**Pull request by ahsimb**
closes #45

### CODE REVIEW

File: exasol/python_extension_common/deployment/language_container_builder.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+
+def copy_slc_flavor(dest_dir: str | Path) -> None:
+    """
+    Copies the content of the language_container directory to the specified
+    destination directory.
+    """
+    files = resources.files(__package__).joinpath('language_container')
+    with resources.as_file(files) as pkg_dir:
+        shutil.copytree(pkg_dir, dest_dir, dirs_exist_ok=True)
```
Comment (tkilias) [inline exasol/python_extension_common/deployment/language_container_builder.py:41]: This is not guaranteed to work, depending on how the package is installed. We should have code in the ITDE which extracts resource directories properly.

Comment (ahsimb) [reply exasol/python_extension_common/deployment/language_container_builder.py:41]: It is supposed to be reliable. For example, it should still work if the package is in a zip. The `resources.as_files` extract the content of the resource and copy it into a temporary directory where I can access it using the normal Path.

Comment (tkilias) [reply exasol/python_extension_common/deployment/language_container_builder.py:41]: Ah, misinterpreted it.

File: exasol/python_extension_common/deployment/language_container/flavor_base/language_definition [hunk 2]
```diff
@@ -0,0 +1 @@
+PYTHON3_TE=localzmq+protobuf:///{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}?lang=python#buckets/{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}/exaudf/exaudfclient_py3
```
Comment (tkilias) [inline exasol/python_extension_common/deployment/language_container/flavor_base/language_definition:1]: I think, we should replace PYTHON3_TE with something specified by the user

Comment (ahsimb) [reply exasol/python_extension_common/deployment/language_container/flavor_base/language_definition:1]: Oops, missed that. Thanks for pointing out.

**Review by tkilias — APPROVED** [review]

**Pull request merged**